File: core.shared/Typescript/DTO/V1/Remboursement/remboursementSante.ts
/* Auto Generated */

import { RemboursementSanteDetails } from "./remboursementSanteDetails";

export interface RemboursementSante {
    numeroParticipant: string;
    dateRemboursementComplete: Date;
    dateSoinComplete?: Date;
    dateDeRemboursement: string;
    truncatDateDeRemboursement: string;
    libelle: string;
    montantVerseIpeca: string;
    depense: string;
    ordre: string;
    codesActes: string;
    beneficiairePrenom: string;
    beneficiaireDateNaissance: string;
    autreOrganisme: string;
    securiteSociale: string;
    rac: string;
    archivePdf: string;
    numeroBordereau: string;
    libellePaiement: string;
    referenceReleve: string;
    codeBordereau: string;
    numeroLot: number;
    participant: number;
    familleActe: string;
    beneficiaireNom: string;
    beneficiaireId: number;
    remboursementsSanteDetail: RemboursementSanteDetails[];
}

File: core.shared/Net/DTO/V1/Remboursement/RemboursementSante.cs
namespace Core.Shared.Net.DTO.V1.Remboursement
{
    public class RemboursementSante
    {
        public string NumeroParticipant { get; set; }
        public DateTime DateRemboursementComplete { get; set; }
        public DateTime? DateSoinComplete { get; set; }
        public string DateDeRemboursement { get; set; }
        public string TruncatDateDeRemboursement { get; set; }
        public string Libelle { get; set; }
        public string MontantVerseIpeca { get; set; }
        public string Depense { get; set; }
        public string Ordre { get; set; }
        public string CodesActes { get; set; }
        public string BeneficiairePrenom { get; set; }
        public string BeneficiaireDateNaissance { get; set; }
        public string AutreOrganisme { get; set; }
        public string SecuriteSociale { get; set; }
        public string Rac { get; set; }
        public string ArchivePdf { get; set; }
        public string NumeroBordereau { get; set; }
        public string libellePaiement { get; set; }
        public string ReferenceReleve { get; set; }
        public string CodeBordereau { get; set; }
        public int NumeroLot { get; set; }
        public int Participant { get; set; }
        public string FamilleActe { get; set; }
        public string BeneficiaireNom { get; set; }
        public int BeneficiaireId { get; set; }
        public IList<RemboursementSanteDetails> RemboursementsSanteDetail { get; set; }
    }
}
